job-dsls/src/main/resources/job-scripts/prod_osl_trigger_umb_message.jenkinsfile

```
@@ -0,0 +1,141 @@
+import groovy.json.JsonSlurper
+import java.net.URLEncoder
+
+def projects = ["kiegroup/drools", "kiegroup/kogito-runtimes"] // Projects to find the productized version in a milestone
```

I would recommend to follow the same approach we use for the different jobs we have and to be able to start the job using input parameters like https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/6c48ef5c710eaa8beba6a4874f0e597f93a349fa/.ci/jenkins/Jenkinsfile.buildchain#L3
```suggestion
def projects = env.PROJECTS?.trim() ? PROJECTS.split(',') : ["kiegroup/drools", "kiegroup/kogito-runtimes"] // Projects to find the productized version in a milestone
```